prog/base.rb

```diff
@@ -86,6 +86,9 @@ def pop(*args)
     else
       fail "BUG: expect no stacks exceeding depth 1 with no back-link" if strand.stack.length > 1
 
+      pg = Page.from_tag_parts(strand.id, strand.prog, strand.stack.first["deadline_target"])
```

Comment:
nit: Side effect of working on postgres too much 😄 
```suggestion
      page = Page.from_tag_parts(strand.id, strand.prog, strand.stack.first["deadline_target"])
```

Reply:
This was intentional, in other branch we also use pg for Page. (even ubid prefix of Page is pg (which is unfortunate because I would like to use that prefix for postgres :) ))